packages/text-field/icon/index.tsx

```diff
@@ -75,9 +73,7 @@ export default class Icon extends React.Component<
   }
 
   componentWillUnmount() {
-    if (this.foundation_) {
-      this.foundation_.destroy();
-    }
+    this.foundation.destroy();
```

Comment:
I know componentWillUnmount is executed after componentDidMount.
Is null check needed?

Reply:
I realize that, and I'm trying to remember why we did that. I also might be getting confused with something. Since we're inconsistent with this, please keep this change.